components/inputs/docs/input-checkbox.md

```diff
@@ -1,51 +1,78 @@
+* Use an indeterminate state to indicate a mixed state where some child items are checked and some are not.
 * Use as progressive disclosure in forms, so long as users are made aware both visually and non-visually that new options have been made available.
 <!-- docs: end dos -->
 
 <!-- docs: start donts -->
 * Don't use as a toggle that performs an immediate action, use a Switch component.
-* Don't use for mutually exclusive options, use radio buttons.
+* Don't use for mutually exclusive options, use radio input groups.
 * Don't use labels as “instructions” or “phrases”. Good label: “Visible to Students”. Bad label: (“Check this to make it visible to students”)
 * Don't use labels to describe the default or “off” state of the option. As much as possible, use the label to refer to the “on” state. Good label: “Visible”. Bad label: “Hidden”.
 <!-- docs: end donts -->
 <!-- docs: end best practices -->
 
+## Checkbox Input Group [d2l-input-checkbox-group]
+
+When multiple checkboxes are displayed together, wrap them in a `<d2l-input-checkbox-group>`.
+
+This will not only provide consistent spacing between checkboxes, but the group also internally renders a `<fieldset>` and `<legend>` with the provided `label`, which gives additional accessibility context. The label can be hidden visually if desired.
+
+<!-- docs: demo code properties name:d2l-input-checkbox-group sandboxTitle:'Checkbox Group' display:block -->
+```html
+<script type="module">
+  import '@brightspace-ui/core/components/inputs/input-checkbox.js';
+  import '@brightspace-ui/core/components/inputs/input-checkbox-group.js';
+</script>
+<d2l-input-checkbox-group label="Toppings">
+  <d2l-input-checkbox label="Ketchup"></d2l-input-checkbox>
+  <d2l-input-checkbox label="Mustard"></d2l-input-checkbox>
+  <d2l-input-checkbox label="Relish"></d2l-input-checkbox>
+</d2l-input-checkbox-group>
+```
+
+<!-- docs: start hidden content -->
+### Properties
+
+| Property | Type | Description |
+|---|---|---|
+| `label` | String, required | Label for the group of checkboxes |
+| `label-hidden` | Boolean | Hides the label visually |
+<!-- docs: end hidden content -->
+
 ## Checkbox Input [d2l-input-checkbox]
 
-The `<d2l-input-checkbox>` element can be used to get a checkbox and optional visible label.
+The `<d2l-input-checkbox>` element can be used on its own or as an option within a `<d2l-input-checkbox-group>` collection. It will display a checkbox and optional visible label.
+
+When provided with a `name`, the checkbox can participate in forms when it is `checked` and enabled. Its value will be `"on"` unless another is provided using the `value` attribute.
 
 <!-- docs: demo code properties name:d2l-input-checkbox sandboxTitle:'Checkbox Input' display:block -->
 ```html
 <script type="module">
   import '@brightspace-ui/core/components/inputs/input-checkbox.js';
-
-  window.addEventListener('load', () => {
```

Comment:
This didn't seem particularly helpful and we don't normally provide this kind of code sample.